website/source/docs/provisioners/shell.html.md

```diff
@@ -78,6 +78,13 @@ Optional parameters:
     **Important:** If you customize this, be sure to include something like the
     `-e` flag, otherwise individual steps failing won't fail the provisioner.
 
+-   `remote_folder` (string) - The folder where the script will be uploaded
+    to in the machine. This defaults to '/tmp'. The value must be a directory
+    that already exists.
+
+-   `remote_file` (string) - The filename the uploaded script will have on the machine.
+    This defaults to 'script_nnn.sh'.
+
 -   `remote_path` (string) - The filename where the script will be uploaded
```

Comment:
This now has the same documentation as `remote_file`, can you include some language to distinguish them ("full path to the file") and also what happens if all three are specified?